tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "lib": ["es6", "dom"],

```

Comment:
dom? do we want dom? i feel like we do not want dom.

Reply:
https://github.com/Microsoft/TypeScript/wiki/What's-new-in-TypeScript#including-built-in-type-declarations-with---lib